### PR TITLE
Update cyberark readme

### PR DESCRIPTION
**Pull request by cherryleaf**
Added link to Cyberark's documentation

Type of change

- Enhancement

## What does this PR do?

Added link to Cyberark's documentation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/cyberark/docs/README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # CyberArk integration
 
-This integration is for CyberArk device's logs. It includes the following
+This integration is for [CyberArk](https://docs.cyberark.com/Product-Doc/OnlineHelp/Portal/Content/Resources/_TopNav/cc_Portal.htm) device's logs. It includes the following
 datasets for receiving logs over syslog or read from a file:
+
 - `corepas` dataset: supports CyberArk logs.
```
Comment (r00tu53r) [inline packages/cyberark/docs/README.md:6]: We could also update the documentation link for the dataset pointing to the particular product. In this case for `corepas` it is [Privileged Access Manager](https://docs.cyberark.com/Product-Doc/OnlineHelp/PAS/Latest/en/Content/Resources/_TopNav/cc_Home.htm).

`corepas` dataset: supports ingesting logs from CyberArk Privileged Access Manager

Comment (cherryleaf) [reply packages/cyberark/docs/README.md:6]: I've added the link

**Comment by ellis-elastic** [comment]
Moved under ellis-elastic #3536

**Pull request closed without merging**